gradle/libs.versions.toml

```diff
@@ -11,6 +11,7 @@ androidSdk-target = "34"
 androidSdk-automotive-min = "29"
 androidSdk-wear-min = "26"
 androidSdk-wear-target = "33"
+androidTools = "31.7.3"
```

Comment:
I cannot find a changelog on Google's sites, but the latest version appears to be [31.10.0](https://mvnrepository.com/artifact/com.android.tools.lint/lint-api).